language/e2e-tests/src/tests/admin_script.rs

```diff
@@ -77,3 +78,54 @@ fn admin_script_rotate_key() {
 
     assert_eq!(updated_sender.authentication_key(), new_key_hash.as_slice());
 }
+
+#[test]
+fn admin_script_rotate_key_multi_signer() {
+    let mut executor = FakeExecutor::from_genesis_file();
+    let new_account = AccountData::new(100_000, 0);
+    executor.add_account_data(&new_account);
+
+    let privkey = Ed25519PrivateKey::generate_for_testing();
+    let pubkey = privkey.public_key();
+    let new_key_hash = AuthenticationKey::ed25519(&pubkey).to_vec();
+
+    let script_body = {
+        let code = "
+    import 0x1.LibraAccount;
+
+    main(account: &signer, auth_key_prefix: vector<u8>) {
```

Comment:
why does this take a single signer (and above double signer)?

Reply:
This is a test to make sure that it fails if it only uses one signer when it should take two.